src/nunit-gui/Views/XmlView.cs

```diff
@@ -45,6 +45,9 @@ public XmlView()
             InitializeComponent();
 
             XmlPanel = new ControlElement<Panel>(xmlPanel);
+            selectAllToolStripMenuItem.Click += (s, a) => xmlTextBox.SelectAll();
+            copyToolStripMenuItem.Click += (s, a) => xmlTextBox.Copy();
+            wordWrapToolStripMenuItem.CheckedChanged += (s, a) => xmlTextBox.WordWrap = wordWrapToolStripMenuItem.Checked;
```

Comment:
Just noting that you are not going through the presenter here. If we wanted to test the behavior, we would have to feed the events to the presenter through some kind of ViewElement. Then NSubstitute would work for us in mocking the view. Don't change it now, however, since we may change the architecture before we are done.